esi_leap/objects/contract.py

```diff
@@ -99,3 +118,22 @@ def expire(self, context=None):
         # expire contract
         self.status = statuses.EXPIRED
         self.save(context)
+
+    # @classmethod
+    # def split(cls, context, c):
+    #     # split contract with multiple offers into serval contracts
+    #     offers = c['offers']
+    #     contracts = []
+    #     for o_uuid in offers:
+    #         contract_dict = {}
+    #         o = esi_leap.objects.offer.Offer.get(context, o_uuid).to_dict()
+    #         contract_dict['offer_uuid'] = o['uuid']
+    #         contract_dict['project_id'] = o['project_id']
+    #         contract_dict['start_date'] = c['start_time']
+    #         contract_dict['end_date'] = c['end_time']
+    #         contract_dict['status'] = c['status']
+    #         contract_dict['cost'] = o['properties'].get('floor_price', 0)
+    #         contract_dict['created_at'] = c['time_created']
+    #         contract_dict['updated_at'] = None
+    #         contracts.append(contract_dict)
+    #     return contracts
```

Comment:
This can probably all just be removed then.

Reply:
I think common/flocx_market.py may have been accidentally left out of the PR?

Reply:
ops, I missed that file. Already added it